src/component/page/gjennomforing-detaljer/deltaker-oversikt/SorterbarKolonneHeader.tsx

```diff
@@ -30,7 +31,10 @@ export const SorterbarKolonneHeader = (props: SortableHeaderProps) : JSX.Element
 			<button
 				className={styles.header}
 				aria-label={ariaLabel}
-				onClick={() => setDeltakerSortering({ type: nesteSorteringType, kolonne: kolonne })}
+				onClick={() => {
+					setDeltakerSortering({ type: nesteSorteringType, kolonne: kolonne })
+					loggKlikk('SorterKolonne', kolonne, nesteSorteringType)
```

Comment:
Burde vi heller ha en spisset funksjon for logging av sortering som igjen kaller loggKlikk.
Kunne også vurdert å logge sortering separat fra klikking med mer egnede navn på feltene,

Reply:
Ser ikke helt at det er noe å tjene på å lage en funksjon i dette tilfelle